src/theme.ts

```diff
@@ -160,6 +165,7 @@ let White = '#FFFFFF';
 // Dev Tool coloring - Colors for development and testing purposes
 let TestTool = '#FF8400';
 let TestToolContrast = '#FF0000';
+let TestToolHalfOpacity = '#FF840082';
```

Comment:
Does this have to do with the issue we saw on Evan's debugging demo where defaulted dark-mode users will see these colors by default?

Reply:
If so, I would prefer if these colors were pointed towards default until these colors have been finalized or are not orange

Reply:
This is just used on the profile page, in front of the profile image.  I just made a halfOpacity color for testing, I can make it default if needed since dark mode isn't used currently @amos-cha...

Reply:
Well, since I've pushed the fix to add some actual dark mode colors (it's in a separate PR and will be included with other review changes), this is now unused (I'll leave the variable in for our testing purposes for now @amos-cha .